JavaScript/JavaScriptSDK.Tests/CheckinTests/SendBuffer.tests.ts

```diff
@@ -0,0 +1,420 @@
+﻿/// <reference path="..\TestFramework\Common.ts" />
+/// <reference path="../../JavaScriptSDK/sender.ts" />
+/// <reference path="../../javascriptsdk/appinsights.ts" />
+
+class SendBufferTests extends TestClass {
+
+    private getArraySendBuffer: (emitLineDelimitedJson?: boolean) => Microsoft.ApplicationInsights.ArraySendBuffer;
+    private getSessionStorageSendBuffer: (emitLineDelimitedJson?: boolean) => Microsoft.ApplicationInsights.SessionStorageSendBuffer;
+
+    public testCleanup() {
+        Microsoft.ApplicationInsights.DataLossAnalyzer.enabled = false;
```

Comment:
You were removing usage of this in a prior pull request but I presume the code is staying for potential future re-use?